charts/openbao/README.md

```diff
@@ -165,8 +165,6 @@ Kubernetes: `>= 1.27.0-0`
 | server.dev.devRootToken | string | `"root"` |  |
 | server.dev.enabled | bool | `false` |  |
 | server.enabled | string | `"-"` |  |
-| server.enterpriseLicense.secretKey | string | `"license"` |  |
```

Comment:
If you remove this option here, can you please also remove it from the `values.yaml` file and any templates it is used in?